build.gradle

```diff
@@ -343,6 +343,8 @@ subprojects {
     classpath = sourceSets.integrationTest.runtimeClasspath
     outputs.upToDateWhen { false }
 
+    jvmArgs = ['-Xms512m', '-Xmx1g']
```

Comment:
The CircleCI `config.yml` also setting JVM args https://github.com/PegaSysEng/ethsigner/blob/1d4de2405d733ecf0de550f0b93e21fcd7ceb6aa/.circleci/config.yml#L10

How does that interact with defining inside the Gradle file, which ones get overridden?

Reply:
My reading of https://docs.gradle.org/current/userguide/build_environment.html#sec:configuring_jvm_memory is that individual test targets require their own args.  My own testing seems to confirm this.

Any insight?

Reply:
The next line in config that file https://github.com/PegaSysEng/ethsigner/blob/1d4de2405d733ecf0de550f0b93e21fcd7ceb6aa/.circleci/config.yml#L11 sets using the gradle launch option that covers configuring the build JVM.

> Certain tasks, like the test task, also fork additional JVM processes. You can configure these through the tasks themselves. They all use -Xmx512m by default.

That suggests it depends on the forking model applied in JUnit (which is configurable)

Reply:
_If_ you intend to allow 1g of memory per a test fork ...how many of these forks are you wanting/able to support?